lib/workos/configuration.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+# typed: false
+
+module WorkOS
+  # Configuration class sets config initializer
+  class Configuration
+    attr_accessor :timeout, :key
+
+    def initialize
+      @timeout = timeout || 60
+      @key = key
```

Comment:
I think we can remove the calls to the `key` and `timeout` attr readers, since this is happening inside of `initialize` where it wouldn't be possible for them to have a value other than `nil` yet:

```suggestion
      @timeout = 60
```

I think we can also omit the explicit initialization of `key`, since there isn't a reasonable default for it.